src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (valueResult) {
+      return function (value) {
+        return valueResult + value;
```

Comment:
[NAMING] - The parameter `valueResult` is not descriptive of its purpose. Consider renaming it to `currentResult` or simply `result` to reflect that it represents the current result of the calculator.